app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/showscreen/ShowScreen.kt

```diff
@@ -0,0 +1,671 @@
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.FilterChip
+import androidx.compose.material3.FilterChipDefaults
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.zIndex
+import coil.compose.AsyncImage
+import hr.dice.filipbionda.tmdbpractice.R
+import hr.dice.filipbionda.tmdbpractice.data.mocks.mockMovies
+import hr.dice.filipbionda.tmdbpractice.data.models.Actor
+import hr.dice.filipbionda.tmdbpractice.data.models.MediaItem
+import hr.dice.filipbionda.tmdbpractice.ui.components.ExpandedButton
+import hr.dice.filipbionda.tmdbpractice.ui.theme.TMDBPracticeTheme
+import hr.dice.filipbionda.tmdbpractice.ui.theme.white_CC
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+val mockMovie = MediaItem(
+    title = "Inception",
+    rating = 87,
+    coverPath = "https://im.jdmagicbox.com/comp/jd_social/news/2021dec11/image-1092864-84qgkwd5wz.jpg?clr=#382e38",
+    date = "21/07/2022",
+    language = "EN",
+    overview = "A thief who steals corporate secrets through the use of dream-sharing technology is given the inverse task of planting an idea into the mind of a C.E.O.",
+    actors = listOf(
+        Actor(name = "Leonardo", surname = "DiCaprio" ,imagePath = "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS8yhDDv3b5ixR0-eeKOz9sfth6qlJxUFRuMk3fRJktRUVlO4fJ"),
+        Actor(name = "Joseph", surname = "Gordon-Levitt", imagePath = "https://m.media-amazon.com/images/M/MV5BMTY3NTk0NDI3Ml5BMl5BanBnXkFtZTgwNDA3NjY0MjE@._V1_.jpg"),
+        Actor(name = "Ellen", surname = "Page", imagePath = "https://resizing.flixster.com/VsatQP6O9THxjHuKRsWwXjtPBAg=/fit-in/352x330/v2/http://media.baselineresearch.com/images/1260564/1260564_full.jpg")
+    ),
+    categories = listOf("Action", "Sci-Fi", "Thriller", "Anime", "Sports", "Specials"),
+    duration = 140,
+    seasons = null,
+    trailerUrl = ""
+)
+
+@Composable
+fun ShowScreen(
+    mediaItem: MediaItem,
+    navigateBack: () -> Unit,
+    openRecommendedMediaItem: (MediaItem) -> Unit,
+    playTrailer: (String) -> Unit ,
+    modifier: Modifier = Modifier
+){
+    Box(
+        modifier = modifier
+            .fillMaxSize()
```

Comment:
Modifier passed from the parameter shouldn't provide size in the composable itself but rather in the calling site

Reply:
Ok.